### PR TITLE
[chamberlainmyq] Added new Close Error and Open Error Switch Items fo…

### PR DESCRIPTION
**Pull request by ofirster**
MyQ Garage Door may have a problem that prevents the door from being opened or closed. This is described in Chamberlain's website: [How to Resolve a Close Error in the myQ App
](https://support.chamberlaingroup.com/s/article/How-to-resolve-a-Close-error-in-the-myQ-app)
When the problem occurs, we would like to inform the user that there is a problem with the door.
The purpose of this Pull Request is to expose two new OpenHAB Switch Items:

1. closeError
2. openError

Each of the items will be ON when the error occurs in the door, and OFF when it doesn't occur.

The new Items can be used to inform the user that there are problems with the Garage door.

### CODE REVIEW

File: bundles/org.openhab.binding.myq/README.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ Once an account has been added, garage doors and lamps will automatically be dis
 | Channel       | Item Type     | Thing Type       | States                                                 |
 |---------------|---------------|------------------|--------------------------------------------------------|
 | status        | String        | garagedoor       | opening, closed, closing, stopped, transition, unknown |
-| rollershutter | Rollershutter | garagedoor       | UP, DOWN, 0%, 100%                                     |
+| rollershutter | Rollershutter | garagedoor       | UP, DOWN, 0%, 100%     
+| closeError    | Switch        | garagedoor       | ON (has error), OFF (doesn't have error)
+| openError     | Switch        | garagedoor       | ON (has error), OFF (doesn't have error)|
 | switch        | Switch        | garagedoor, lamp | ON (open), OFF (closed)
```
Comment (digitaldan) [inline bundles/org.openhab.binding.myq/README.md:37]:
```suggestion
| rollershutter | Rollershutter | garagedoor       | UP, DOWN, 0%, 100%                                     |
| closeError    | Switch        | garagedoor       | ON (has error), OFF (doesn't have error)               |
| openError     | Switch        | garagedoor       | ON (has error), OFF (doesn't have error)               |
| switch        | Switch        | garagedoor, lamp | ON (open), OFF (closed)                                
```

File: bundles/org.openhab.binding.myq/README.md [hunk 2]
```diff
@@ -61,7 +65,7 @@ Switch MyQGarageDoorLamp "Lamp [%s]" {channel = "myq:lamp:home:efgh6789:switch"}
 sitemap MyQ label="MyQ Demo Sitemap" {
   Frame label="Garage Door" {
     String item=MyQGarageDoor1Status
-    Switch item=MyQGarageDoor1Switch
+    Switch item=MyQGarageDoor1Switch
```
Comment (digitaldan) [inline bundles/org.openhab.binding.myq/README.md:68]: Extra whitespace
```suggestion
    Switch item=MyQGarageDoor1Switch
```

**Review by digitaldan — CHANGES_REQUESTED** [review]
Thanks @ofirster for the contribution !   I left just a couple small formatting comments.  And thank you for pointing out the MyQ FAQ on how this is used. 

 I am a little on the fence about calling it `closeerror` vs `closeallowed` and not inverting the state to better match the state value and state name in the API, even though the FAQ seems to refer it as a "Close Error".  I prefer to match the OH side up as close as possible to API's that we talk to, it seems to "age" better that way over time.    But.... I also see how this makes more sense for an end user.   I could go either way.

File: bundles/org.openhab.binding.myq/README.md (hunk 1 above)
Comment (digitaldan) [reply bundles/org.openhab.binding.myq/README.md:37]: FYI i use http://markdowntable.com/ to quickly fix tables like this...

**Comment by ofirster** [comment]
> Thanks @ofirster for the contribution ! I left just a couple small formatting comments. And thank you for pointing out the MyQ FAQ on how this is used.
> 
> I am a little on the fence about calling it `closeerror` vs `closeallowed` and not inverting the state to better match the state value and state name in the API, even though the FAQ seems to refer it as a "Close Error". I prefer to match the OH side up as close as possible to API's that we talk to, it seems to "age" better that way over time. But.... I also see how this makes more sense for an end user. I could go either way.

File: bundles/org.openhab.binding.myq/README.md (hunk 1 above)
Comment (ofirster) [reply bundles/org.openhab.binding.myq/README.md]: @digitaldan, thanks for the information. I will use this website next time 👍

**Comment by ofirster** [comment]
> Thanks @ofirster for the contribution ! I left just a couple small formatting comments. And thank you for pointing out the MyQ FAQ on how this is used.
> 
> I am a little on the fence about calling it `closeerror` vs `closeallowed` and not inverting the state to better match the state value and state name in the API, even though the FAQ seems to refer it as a "Close Error". I prefer to match the OH side up as close as possible to API's that we talk to, it seems to "age" better that way over time. But.... I also see how this makes more sense for an end user. I could go either way.

@digitaldan, you brought an interesting topic to discuss. I agree with you that matching to the API is very important, but on the other hand, it is clearer for the user that he has an error with the door, if we go with Close Error and Open Error.
I'm glad you agree to go with both ways, so I would like to leave it as it stands for now.
Thank you 🥇

**Comment by ofirster** [comment]
Thanks @digitaldan for the requested changes, I committed them.
Also, I uploaded a release of the JAR file for testing.
[Close Error and Open Error Release](https://github.com/ofirster/openhab-addons/releases/tag/myq-closeerror-open-error)

**Review by ofirster — COMMENTED** [review]
Added Close Error and Open Error Items

File: bundles/org.openhab.binding.myq/README.md [hunk 3]
```diff
@@ -32,7 +32,9 @@ Once an account has been added, garage doors and lamps will automatically be dis
 |---------------|---------------|------------------|--------------------------------------------------------|
 | status        | String        | garagedoor       | opening, closed, closing, stopped, transition, unknown |
 | rollershutter | Rollershutter | garagedoor       | UP, DOWN, 0%, 100%                                     |
-| switch        | Switch        | garagedoor, lamp | ON (open), OFF (closed)
+| closeError    | Switch        | garagedoor       | ON (has error), OFF (doesn't have error)               |
+| openError     | Switch        | garagedoor       | ON (has error), OFF (doesn't have error)               |
+| switch        | Switch        | garagedoor, lamp | ON (open), OFF (closed)
```
Comment (digitaldan) [inline bundles/org.openhab.binding.myq/README.md:37]:
```suggestion
| switch        | Switch        | garagedoor, lamp | ON (open), OFF (closed)                                |
```

Comment (digitaldan) [reply bundles/org.openhab.binding.myq/README.md:37]: One last change, i think my copy/paste truncated the last `|` in my previous suggestion

**Review by digitaldan — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.myq/README.md (hunk 3 above)
Comment (ofirster) [reply bundles/org.openhab.binding.myq/README.md:37]: @digitaldan, thanks!

Comment (ofirster) [reply bundles/org.openhab.binding.myq/README.md:37]: @digitaldan I committed your suggestion

**Comment by digitaldan** [comment]
So for some reason the the last `|` in that table keeps getting dropped even though its in the suggested change (so not a copy and paste error).  Very strange.  Can you add it to your local file and do a regular push ?   Not sure why the Github suggestions are not working here.

**Comment by ofirster** [comment]
> So for some reason the last `|` in that table keeps getting dropped even though its in the suggested change (so not a copy and paste error). Very strange. Can you add it to your local file and do a regular push ? Not sure why the Github suggestions are not working here.

@digitaldan, wow I head some real difficulties with signing off a specific commit.
DCO is okay now.
Do we have more changes to make?

**Review by digitaldan — APPROVED** [review]
LGTM , thanks!

**Comment by ofirster** [comment]
@digitaldan, is it likely to be merged and released?

**Comment by kaikreuzer** [comment]
As @digitaldan already approved it, I guess it can indeed be merged!

**Pull request merged**